portafly/src/components/data-list/SearchWidget.tsx

```diff
@@ -97,6 +100,7 @@ interface ISearchBar {
 }
 
 const SearchBar = ({ textInputRef, category, onSearchClick }: ISearchBar) => {
+  const { t } = useTranslation('audienceAccountsListing')
```

Comment:
I think here this is too specific, since the SearchBar component will be used in many other data lists than accounts. If we can't find a more generic set of translations and scope, we might want to pass it in the props maybe?

Reply:
I agree, however that is something we should address in another PR (a restructure of translations). Certainly these strings belong in "data-list" or at least "shared". Maybe I could do that change as part of the DevAccounts list. These strings should not have been merged to the milestone (but the accounts listing PR) in the first place.